### ISSUE TITLE
Issue starting Bazarr with Python3.6

### ISSUE THREAD

**Issue by jaretclifton** [morpheus65535/bazarr#681]
**Describe the bug**
Unable to start Bazarr using Python 3.6

**To Reproduce**
/usr/bin/python3.6 bazarr.py
Bazarr starting...
Traceback (most recent call last):
  File "bazarr/main.py", line 36, in <module>
    from init import *
  File "/opt/bazarr/bazarr/init.py", line 79, in <module>
    from database import db_upgrade
  File "/opt/bazarr/bazarr/database.py", line 6, in <module>
    from helper import path_replace, path_replace_movie, path_replace_reverse, path_replace_reverse_movie
  File "/opt/bazarr/bazarr/helper.py", line 208, in <module>
    sonarr_path_mapping_regex()
  File "/opt/bazarr/bazarr/helper.py", line 30, in sonarr_path_mapping_regex
    sonarr_regex = re.compile("|".join(path_mapping.keys()))
  File "/usr/lib/python3.6/re.py", line 233, in compile
    return _compile(pattern, flags)
  File "/usr/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 562, in compile
    p = sre_parse.parse(p, flags)
  File "/usr/lib/python3.6/sre_parse.py", line 855, in parse
    p = _parse_sub(source, pattern, flags & SRE_FLAG_VERBOSE, 0)

**Expected behavior**
Bazarr should start as expected since the message in the UI says it is now compatible with Python3.6 and later

**Software (please complete the following information):**
 - Bazarr: v0.8.3.4
 - OS: Ubuntu 18.04

**Comment by morpheus65535** [comment]
Python 3.6.x? What is the minor version number?

**Comment by jaretclifton** [comment]
Python 3.6.9

**Comment by morpheus65535** [comment]
I cannot reproduce your issue with 3.6.8 (cannot get 3.6.9:
![image](https://user-images.githubusercontent.com/5130500/69773719-97881800-1161-11ea-881a-d54191f3327e.png)

**Comment by jaretclifton** [comment]
Fresh git clone, attempted to start using:

python3 bazarr.py

Received:

File "bazarr.py", line 19
    print "Python 3 isn't supported. Please use Python " + minimum_python_version + " or greater."
                                                       ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Python 3 isn't supported. Please use Python " + minimum_python_version + " or greater.")?

**Comment by morpheus65535** [comment]
Switch to dev branch.

**Comment by jaretclifton** [comment]
Was finally able to get it to launch after:

Renaming old bazarr folder
Cloning the repo
Switching to development branch
Ensuring it was up to date with git pull
Launching with python3 bazarr.py

I was met with (as expected) the basic wizard screen. In order to preserve the existing data, I copied the data directory into the bazarr directory and attempted to launch with python3. I was met with the errors that originally started this issue. So something in that data directory from the previous versions does not like python3.

**Comment by jaretclifton** [comment]
Definitely related to items in the config.ini file. Even blowing it away and restarting and copying/pasting from the previous UI, valid tests pass and then Bazarr attempts to reboot... and fails to launch. Met with original errors. Blowing away the config file again gets Bazarr to start again with the wizard.

**Comment by morpheus65535** [comment]
Please provide the config.ini file so I can take a look at it and try to start an instance using it.

**Comment by jaretclifton** [comment]
`[assrt]
token = 

[sonarr]
ssl = False
apikey = OBFUSCATED
full_update = Daily
full_update_day = 6
ip = OBFUSCATED
full_update_hour = 4
only_monitored = True
port = 8989
base_url = 

[titlovi]
username = 
password = 

[radarr]
ssl = False
apikey = OBFUSCATED
full_update = Daily
full_update_day = 6
ip = OBFUSCATED
full_update_hour = 5
only_monitored = True
port = 7878
base_url = 

[legendastv]
username = 
password = 

[anticaptcha]
anti_captcha_key = 

[auth]
username = OBFUSCATED
password = OBFUSCATED
type = form
apikey = OBFUSCATED

[general]
update_restart = True
movie_default_hi = False
movie_default_language = None
serie_default_forced = False
chmod = 0640
use_scenename = True
use_postprocessing = False
use_radarr = True
base_url = /
page_size = 1000
wanted_search_frequency = 3
multithreading = True
branch = development
use_sonarr = True
serie_default_hi = False
path_mappings_movie = [['\\\\OBFUSCATED\\Storage\\Movies', '/media/fs01/Movies'], ['', ''], ['', ''], ['', ''], ['', '']]
days_to_upgrade_subs = 30
serie_default_language = None
postprocessing_cmd = 
debug = False
utf8_encode = True
adaptive_searching = True
ip = 0.0.0.0
subfolder_custom = 
enabled_providers = opensubtitles,subscene,subscenter,supersubtitles,tvsubtitles
port = 6767
path_mappings = [['\\\\OBFUSCATED\\Storage\\TV Shows', '/media/fs01/TV Shows'], ['', ''], ['', ''], ['', ''], ['', '']]
upgrade_manual = True
anti_captcha_provider = death-by-captcha
minimum_score_movie = 70
upgrade_subs = True
single_language = True
movie_default_forced = False
upgrade_frequency = 12
serie_default_enabled = True
ignore_pgs_subs = True
movie_default_enabled = True
subfolder = current
chmod_enabled = False
auto_update = True
throtteled_providers = {}
minimum_score = 90
use_embedded_subs = True
use_mediainfo = True

[subscene]
username = 
password = 

[analytics]
enabled = True
visitor = OBFUSCATED

[opensubtitles]
username = OBFUSCATED
use_tag_search = False
skip_wrong_fps = False
ssl = True
vip = False
timeout = 15
password = OBFUSCATED

[addic7ed]
username = 
password = 
random_agents = True

[napisy24]
username = 
password = 

[proxy]
username = 
url = 
exclude = localhost,127.0.0.1
password = 
type = None
port = 

[deathbycaptcha]
username = OBFUSCATED
password = OBFUSCATED

[xsubs]
username = 
password = 

[betaseries]
token = 

[titulky]
`

**Comment by jaretclifton** [comment]
Interesting that github escaped the characters. The path_mappings are 4 backslashes, host, two backslashes, path, two backslashes, TV Shows (or Movies)

**Comment by jaretclifton** [comment]
Blowing away the config and putting as bare minimum of config items in the UI, hitting save and restart results in a failure to start Bazarr.

**Comment by morpheus65535** [comment]
Could it be an issue with permission to access the config.ini file? I've saved your config as-is in a config.ini file and I can start Bazarr without any issue. I only changed form auth to None.

**Comment by rodalpho** [comment]
I'm unable to start Bazarr on Python 3.6.9. Given the timer, that's a concern now.

https://pythonclock.org/

```root@host:/opt/bazarr/data/log# python3 --version
Python 3.6.9
root@host:/etc/systemd/system# /usr/bin/python3 /opt/Bazarr/bazarr.py
Traceback (most recent call last):
  File "/opt/Bazarr/bazarr.py", line 15, in <module>
    from bazarr.get_args import args
  File "/opt/Bazarr/bazarr/get_args.py", line 6, in <module>
    from distutils.util import strtobool
ModuleNotFoundError: No module named 'distutils.util'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport/fileutils.py", line 15, in <module>
    from configparser import ConfigParser, NoOptionError, NoSectionError
ImportError: bad magic number in 'configparser': b'\x03\xf3\r\n'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 63, in apport_excepthook
    from apport.fileutils import likely_packaged, get_recent_crashes
  File "/usr/lib/python3/dist-packages/apport/__init__.py", line 5, in <module>
    from apport.report import Report
  File "/usr/lib/python3/dist-packages/apport/report.py", line 30, in <module>
    import apport.fileutils
  File "/usr/lib/python3/dist-packages/apport/fileutils.py", line 19, in <module>
    from ConfigParser import ConfigParser, NoOptionError, NoSectionError
ModuleNotFoundError: No module named 'ConfigParser'

Original exception was:
Traceback (most recent call last):
  File "/opt/Bazarr/bazarr.py", line 15, in <module>
    from bazarr.get_args import args
  File "/opt/Bazarr/bazarr/get_args.py", line 6, in <module>
    from distutils.util import strtobool
ModuleNotFoundError: No module named 'distutils.util'```

**Comment by morpheus65535** [comment]
Is it still an issue?

**Comment by Yaff1e** [comment]
> I had lots of trouble starting Bazarr on python 3.6.9 also. I just fixed them by running
> 
> find . -name *.pyc -delete
> 
> In the Bazarr install directory. Also install pip3 and install the requirements.txt from the wiki.

Deleting the .pyc files fixed it for me on 3.6.9. Thanks!
